sdk/src/main/java/com/horizen/account/api/rpc/handler/RpcHandler.java

```diff
@@ -30,4 +34,29 @@ public ApiResponse apply(RpcRequest request) {
             return new RpcResponseError(request.getId(), RpcError.fromCode(RpcCode.InternalError, e.getMessage()));
         }
     }
+
+    public List<RpcResponse> apply(RpcBatchRequest request) {
+
+        final List<RpcResponse> responseList = new ArrayList<>();
+
+        for(RpcRequest rpcRequest: request.getRpcRequests()) {
+            try {
+                if (rpcService.hasMethod(rpcRequest.getMethod())) {
+                    var result = rpcService.execute(rpcRequest);
+                    responseList.add(new RpcResponseSuccess(rpcRequest.getId(), result));
+                } else
+                    responseList.add(new RpcResponseError(rpcRequest.getId(), RpcError.fromCode(RpcCode.MethodNotFound)));
+            } catch (RpcException e) {
+                responseList.add(new RpcResponseError(rpcRequest.getId(), e.error));
+            } catch (Throwable e) {
+                responseList.add(new RpcResponseError(rpcRequest.getId(), RpcError.fromCode(RpcCode.InternalError, e.getMessage())));
+            }
+        }
+
+        // add invalid requests to the batch response
+        for(int i=0; i<request.getInvalidRequestsNumber(); i++)
+            responseList.add(new RpcResponseError(new RpcId(), RpcError.fromCode(RpcCode.InvalidRequest, null)));
```

Comment:
It is not clear to me this part.
It seems we append invalid requests messages to the end of the result array. And moreover we miss `rpcId` fields.

Instead in case of batch request we should return the results in the same order as requests and with proper id fields.

I guess, we miss some UTs for this case

Reply:
As reported the `AccountEthRpcRoute` pr comment we refactor the previous implementation so right now the `RpcHandler` is handling single request only